src/pages/Instances.jsx

```diff
@@ -11,44 +11,98 @@ import {
 
 import { InstancesData, InstancesGrid } from "../data/dummy";
 import { Header } from "../components";
-import { DropdownAction } from "../components/DropdownAction";
-import { BsPlusLg } from "react-icons/bs";
-import { getUserInstances } from '../api';
+import { BsCloudDownload } from "react-icons/bs";
+import { getUserInstances } from "../api";
+import { updateVM, terminateVM } from "../api/vm";
 import { Link } from "react-router-dom";
+import SSHCert from "../components/SSHCert";
+import DropdownAction from "../components/DropdownAction";
 
 const Instances = () => {
-
   const [data, setData] = useState([]);
+  // const [actionStatus, setActionStatus] = useState({});
   const toolbarOptions = ["Search"];
 
   const editing = { allowDeleting: true, allowEditing: true };
   const settings = { wrapMode: "Content" };
 
+  function flattenObject(ob) {
+    const result = {};
```

Comment:
if you think this function will be reused a lot, consider define it somewhere like "util"

Reply:
Fixed, thank you!